### ISSUE TITLE
Write GitHub action to build documentation

### ISSUE THREAD

**Issue by jrsmith3** [jrsmith3/ibei#55]
# Overview
The scope of this ticket is to write a GitHub action to run the `tox` action that builds the package documentation and uploads it as an artifact to the action run.

# Related issues
* Depends on #54.

**Comment by jrsmith3** [comment]
I re-scoped this issue for the action to only build the documentation. I just realized that I have a `tox` action to actually build the documentation, so this split may not be as useful as I initially thought. 🤔

**Comment by jrsmith3** [comment]
Superseded by #79.
